WildFly_27.0.0.Final/jakarta-full-platform-jdk17.adoc

```diff
@@ -0,0 +1,557 @@
+= TCK Results
```

Comment:
This folder should be WildFly_27.0.0.Alpha5

Reply:
fixed